src/engine/enginevumeter.cpp

```diff
@@ -93,6 +99,24 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
     } else {
         --m_peakDuration;
     }
+
+    if (clipped & 1) {
```

Comment:
Well, this is redundant because:  (clipped >0) == (clipped&1) || (clipped&2). What about copying the same comment I wrote in sampleutil.h here?:

> returns >0 in case of clipping (> +-1)
    // The returned int is a bitmask: if the first bit is set the L
    // channel is clipping, if the second bit is set, the R channel does.